test/integration/rise-storage.html

```diff
@@ -354,25 +353,6 @@
           done();
         });
 
-        test("should retry storage request after rise-storage-api-error is encountered", function(done) {
```

Comment:
@donnapep These scenarios rely on `_startTimer` and `refresh` value. If RC is not running, we don't refresh, so it impacts any scenario involving reliance on refreshing. That's why I had to remove due to them failing and not applicable anymore.